package.json

```diff
@@ -7,7 +7,13 @@
     "@fortawesome/fontawesome-free": "^6.2.0",
     "@mapbox/gist-map-browser": "0.2.1",
     "@mapbox/github-file-browser": "0.6.1",
+    "@mapbox/mapbox-gl-draw": "^1.3.0",
+    "@mapbox/mapbox-gl-geocoder": "^5.0.1",
     "@popperjs/core": "^2.11.6",
+    "@turf/area": "^6.5.0",
+    "@turf/bbox": "^6.5.0",
+    "@turf/circle": "^6.5.0",
+    "@turf/length": "^6.5.0",
```

Comment:
Great call here. I think this will be a noticeable savings.

Reply:
Where/when was the old turf metapackage removed from `package.json`? Or was it always bundled elsewhere?

Reply:
Turf was being brought in via a CDN link https://github.com/mapbox/geojson.io/pull/761/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L32

Reply:
Ahhhhhh.